tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -561,42 +561,31 @@ public function forUserTheSpaceSharesShouldContainTheseEntries(string $user, str
 	}
 
 	/**
+	 * @When user :user disables sync for share :share using the Graph API
 	 *
 	 * @param string $user
-	 * @param string $itemId
-	 * @param string $shareSpaceId
 	 *
-	 * @return ResponseInterface
+	 * @return void
+	 * @throws Exception
 	 */
-	public function unmountShare(string $user, string $itemId, string $shareSpaceId): ResponseInterface {
-		return GraphHelper::unmountShare(
+	public function userUnmountsShareUsingTheGraphApi(string $user):void {
```

Comment:
Alos rename the methods name
```suggestion
	public function userDisablesSyncOfShareUsingTheGraphApi(string $user):void {
```